installer/install.iss

```diff
@@ -1834,6 +1858,50 @@ begin
     end;
     EditorSelectionChanged(NIL);
 
+    (*
+     * Create a custom page for modifying the default branch.
+     *)
+
+    DefaultBranchPage:=CreatePage(PrevPageID,'Adjusting the name of the initial branch in new repositories','What would you like Git to name the initial branch after "git init"?',TabOrder,Top,Left);
+
+    // 1st choice
+    RdbDefaultBranch[DB_Unspecified]:=CreateRadioButton(DefaultBranchPage,'Let Git decide','Let Git use its default branch name (currently: "master") for the initial branch'+#13+'in newly created repositories. The Git project <A HREF=https://sfconservancy.org/news/2020/jun/23/gitbranchname/>intends</A> to change this default to'+#13+'a more inclusive name in the near future.',TabOrder,Top,Left);
```

Comment:
It looked like your `currently: "master")` was getting cut off in the picture. How can we make this part get word-wrapped?

Reply:
Nevermind, you fixed that in a later picture.